core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -253,8 +253,16 @@ contract Liquidatable is PricelessPositionManager {
             })
         );
 
+        // Scoping to get rid of a stack too deep error.
         // Adjust the sponsor's remaining position.
-        _reduceSponsorPosition(sponsor, tokensLiquidated, lockedCollateral, withdrawalAmountToRemove);
+        {
+            // Part of the withdrawal request is also removed. Ideally:
+            // liquidatedCollateral + withdrawalAmountToRemove = lockedCollateral.
+            FixedPoint.Unsigned memory withdrawalAmountToRemove = positionToLiquidate.withdrawalRequestAmount.mul(
```

Comment:
Why is this added?

Reply:
It's moved from above.

Reply:
got it. Wonder why the stack too deep error occurs with these changes

Reply:
Because I added new local variable: the function parameter. Hate Solidity for these problems.

Reply:
Sucks :/